formats/shared/default_properties/all.jsonnet

```diff
@@ -23,6 +23,10 @@
       "live",
     ],
   },
+  phase_message: {
+    description: "The message associated with the design phase for this content item",
+    type: "string",
```

Comment:
Rather than a string I think this is likely to contain either Govspeak or HTML. 

The precedent for this is to use the [multiple_content_types definition](https://github.com/alphagov/govuk-content-schemas/blob/7865213ca35962fc5ef80dd90b8652de0e5799b2/formats/shared/definitions/publishing_api_base.jsonnet#L70-L88)